app/models/Team.ts

```diff
@@ -27,6 +27,10 @@ class Team extends BaseModel {
   @observable
   documentEmbeds: boolean;
 
+  @Field
+  @observable
+  preferredCollectionId: string | null;
```

Comment:
Rename might be a bit of a pain, but better now than later. "Preferred" has some extra connotations in English, "default" is also language we use elsewhere

```suggestion
  defaultCollectionId: string | null;
```